### PR TITLE
Refactor hard coded etherscan links

### PR DESCRIPTION
**Pull request by evgenyboxer**
This PR introduces a reusable util (initially just for etherscan) for building links for etherscan (tx/address), given a networkId.

### CODE REVIEW

File: src/utils/explorers.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { SUPPORTED_NETWORKS, isMainNet } from '../utils/networkUtils';
+
+const getEtherScanBaseURL = networkId => {
+	const network = SUPPORTED_NETWORKS[networkId];
+
+	if (isMainNet(networkId) || network == null) {
+		return 'https://etherscan.io';
```
Comment (evgenyboxer) [inline src/utils/explorers.js]: @clementbalestrat I wasn't sure what do we want to return when the network is invalid, so I just default it to MainNet. Let me know if you want this changed.

File: src/ducks/wallet.js [hunk 2]
```diff
@@ -15,6 +15,8 @@ const defaultState = {
 	currentWallet: null,
 	derivationPath: localStorage.getItem('derivationPath'),
 	balances: null,
+	networkId: null,
+	networkName: null,
```
Comment (evgenyboxer) [inline src/ducks/wallet.js]: I saw these were missing from the wallet info.

File: src/utils/networkUtils.js [hunk 3]
```diff
@@ -35,7 +38,7 @@ export const hasWeb3 = () => {
 	return window.web3;
 };
 
-const defaultNetwork = { name: 'MAINNET', networkId: '1' };
+export const defaultNetwork = { name: 'MAINNET', networkId: '1' };
```
Comment (evgenyboxer) [inline src/utils/networkUtils.js]: Should the defaultNetwork used in the `wallet` reducer as the initial value of `networkName` and `networkId`? This is why I exported it. just wanted to double check.

File: src/utils/explorers.spec.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import { getEtherscanTxLink, getEtherscanAddressLink } from './explorers';
+
+describe('Explorers', () => {
```
Comment (justinjmoses) [inline src/utils/explorers.spec.js]: :pray:

File: src/utils/explorers.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	}
+
+	return `https://${network.toLowerCase()}.etherscan.io`;
+};
+
+export const getEtherscanTxLink = (networkId, txId) => {
```
Comment (justinjmoses) [inline src/utils/explorers.js]: might be worthwhile supporting both network by name and network by ID. You could then make this a standalone module that we can pull into https://github.com/Synthetixio/synthetix as well

Comment (evgenyboxer) [reply src/utils/explorers.js]: Sure, will add support for it. Can we first add it here, and then I'll export it into a module?

File: src/utils/explorers.js (hunk 1 above)
Comment (clementbalestrat) [reply src/utils/explorers.js]: Yeah that sounds good for now!

**Pull request merged**